src/graphics/Renderer.h

```diff
@@ -145,59 +144,65 @@ namespace Graphics {
 
 		virtual bool ReloadShaders() = 0;
 
-		// our own matrix stack
-		// XXX state must die
-		virtual const matrix4x4f &GetCurrentModelView() const = 0;
-		virtual const matrix4x4f &GetCurrentProjection() const = 0;
-		virtual void GetCurrentViewport(Sint32 *vp) const = 0;
-
-		// XXX all quite GL specific. state must die!
-		virtual void SetMatrixMode(MatrixMode mm) = 0;
-		virtual void PushMatrix() = 0;
-		virtual void PopMatrix() = 0;
-		virtual void LoadIdentity() = 0;
-		virtual void LoadMatrix(const matrix4x4f &m) = 0;
-		virtual void Translate(const float x, const float y, const float z) = 0;
-		virtual void Scale(const float x, const float y, const float z) = 0;
+		/// XXX at least use a custom structure you heathens
+		[[deprecated]] virtual void GetCurrentViewport(Sint32 *vp) const = 0;
 
 		// take a ticket representing the current renderer state. when the ticket
 		// is deleted, the renderer state is restored
 		// XXX state must die
-		class StateTicket {
+		class [[deprecated]] StateTicket
```

Comment:
Deprecated?

Reply:
I deprecated the existing methods (to see the extent of their use) before removing them, this is an unintentional slip.